CHANGELOG.md

```diff
@@ -22,6 +23,24 @@ The below sections record the changes for each API version in each (pre-)release
 
 NOTE: SimpleEdgeDiscovery is part of the EdgeCloud API family, and was previously developed within the [EdgeCloud](https://github.com/camaraproject/EdgeCloud) repository. Two releases were made within EdgeCloud, v0.81 and r0.9.3-rc. These releases also include the other EdgeCloud APIs and materials, so the changelog below highlights only the changes related to SimpleEdgeDiscovery. Further information and links to the releases in EdgeCloud can be found below.
 
+# r1.3
+
+This release contains the definition and documentation of
+* simple-edge-discovery v1.0.0, a stable public release.
+
+The API definition(s) are based on
+* Commonalities v0.4.0
+* Identity and Consent Management v0.2.0
+
+It contains the following changes compared to [r1.2 - rc](#r12---rc).
+
+### Changed
+  - info.version to 1.0.0
+  - version in servers.url to v1
+  - API Readiness checklist updated 
+
+Full Changelog: https://github.com/camaraproject/SimpleEdgeDiscovery/compare/r1.2...r1.3
```

Comment:
HI @tanjadegroot  -  unusual situation here is that the last public release (`r0.9.3 -rc` ) was in another repository (EdgeCloud), so I can't make a GitHub URL that directly compares `r1.3` and `r0.9.3 - rc`.

 The majority of the curated changes from 0.9/3 -rc  are noted in the section for r1.1 - rc, with some small changes in r1.2 and r.13. Shall I just add a sentence to explain that?

Reply:
@Kevsy it is not about the GitHub URL but about the list of changes in the release notes for r1.3. As this is the public release, the list of changes should contain all changes since the last public release, not only the ones since the pre-release. Put yourself in the shoes of readers who are only reading the release notes of r1.3 to understand what has changed since the last public release they are using. And yes, that means to repeat the entries from r1.1-rc if they were not reverted later.